pyccel/ast/numpyext.py

```diff
@@ -9,7 +9,7 @@
 
 from .basic          import PyccelAstNode
 from .builtins       import (PythonInt, PythonBool, PythonFloat, PythonTuple,
-                             PythonComplex, PythonReal, PythonImag, PythonList)
+                             PythonComplex, PythonReal, PythonAbs, PythonImag, PythonList)
```

Comment:
Is this a bug fix?

Reply:
Ah sorry, i missed this from #790.

Reply:
This PR should modify only `test_numpy_functions.py`, except the little new line in ccode.py